### PR TITLE
upd(thorium-deb): `117.0.5938.157` -> `119.0.6045.214`

### PR DESCRIPTION
**Pull request by vigress8**

### CODE REVIEW

File: packages/thorium-deb/thorium-deb.pacscript [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+  arm64)
+    url="https://github.com/Alex313031/Thorium-Raspi/releases/download/M${pkgver}/${gives}_${pkgver}_${CARCH}.deb"
+    hash="811dded7557b0ca9a93c3a50e5ce6658f2d185b9858443a76d359485598856b5"
+    ;;
+  i386)
+    url="https://github.com/Alex313031/Thorium-Other/releases/download/${pkgver}-2/${gives}_${pkgver}_${CARCH}.deb"
+    hash="ee5443c8547bb1828e7a105fb5f1ca4eaa43d5a16fd683a37bb1b2e9f57db21c"
+    ;;
+  amd64)
+    if grep -q avx2 /proc/cpuinfo; then
```
Comment (Elsie19) [inline packages/thorium-deb/thorium-deb.pacscript:16]: What's avx?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/thorium-deb/thorium-deb.pacscript (hunk 1 above)
Comment (vigress8) [reply packages/thorium-deb/thorium-deb.pacscript:16]: A CPU feature, google it. Thorium and Mercury provide specialized builds with it

Comment (Elsie19) [reply packages/thorium-deb/thorium-deb.pacscript:16]: I don't like grepping in a pacscript, I *really* want all pacscripts to be simple control flow with no external commands.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/thorium-deb/thorium-deb.pacscript (hunk 1 above)
Comment (vigress8) [reply packages/thorium-deb/thorium-deb.pacscript:16]: Again, if you have a better way I'm all ears. Pacstall does not have a way to specify compatible hardware features

Comment (Elsie19) [reply packages/thorium-deb/thorium-deb.pacscript:16]: Then we should make one! Should we discuss on discord/matrix?

Comment (oklopfer) [inline packages/thorium-deb/thorium-deb.pacscript:16]: The idea with https://github.com/pacstall/pacstall/pull/1025 is you should be able to replace 
```bash
if grep -q avx2 /proc/cpuinfo; then
```
with
```bash
if [[ " ${DCPUFLAGS[*]} " =~ " avx2 " ]]; then
```

**Pull request closed without merging**